### PR TITLE
Update action and blueprints permissions documentation

### PR DESCRIPTION
**Pull request by Tankilevitch**
# Description

What -
Why -
How -

## Type of change

Please leave one option from the following and delete the rest:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Non-breaking change (fix of existing functionality that will not change current behavior)
- [ ] Documentation (added/updated documentation)

### CODE REVIEW

File: docs/resources/port_action.md [hunk 1]
```diff
@@ -223,7 +223,7 @@ Required:
 
 Optional:
 
-- `payload` (String) The Azure Devops workflow payload (array or object encoded to a string)
+- `payload` (String) The Azure Devops workflow [payload](https://docs.getport.io/create-self-service-experiences/setup-backend/#define-the-actions-payload) (array or object encoded to a string use [jsonencode](https://developer.hashicorp.com/terraform/language/functions/jsonencode)). Learn about how to [define the action payload](https://docs.getport.io/create-self-service-experiences/setup-backend/#define-the-actions-payload).
```
Comment (erikzaadi) [inline docs/resources/port_action.md:226]: https://docs.getport.io/create-self-service-experiences/setup-backend/#define-the-actions-payload link is here twice on purpose?
[payload] => learn about how [define]

File: docs/resources/port_action.md [hunk 2]
```diff
@@ -238,7 +238,7 @@ Required:
 Optional:
 
 - `report_workflow_status` (String) Report the workflow status when invoking the action
-- `workflow_inputs` (String) The GitHub workflow inputs (key-value object encoded to a string)
+- `workflow_inputs` (String) The GitHub [workflow inputs](https://docs.getport.io/create-self-service-experiences/setup-backend/#define-the-actions-payload) should be in JSON format, encoded as a string. Use [jsonencode](https://developer.hashicorp.com/terraform/language/functions/jsonencode) to encode arrays or objects. Learn about how to [define the action payload](https://docs.getport.io/create-self-service-experiences/setup-backend/#define-the-actions-payload).
```
Comment (erikzaadi) [inline docs/resources/port_action.md:241]: Ditto 2X Link

File: docs/resources/port_action.md [hunk 3]
```diff
@@ -252,15 +252,15 @@ Required:
...
+- `pipeline_variables` (String) The Gitlab pipeline variables (key-value object encoded to a string use [jsonencode](https://developer.hashicorp.com/terraform/language/functions/jsonencode)). Learn about how to [define the action payload](https://docs.getport.io/create-self-service-experiences/setup-backend/#define-the-actions-payload).
 
 
 <a id="nestedatt--kafka_method"></a>
 ### Nested Schema for `kafka_method`
 
 Optional:
 
-- `payload` (String) The Kafka message payload (array or object encoded to a string)
+- `payload` (String) The Kafka message [payload](https://docs.getport.io/create-self-service-experiences/setup-backend/#define-the-actions-payload) should be in JSON format, encoded as a string. Use [jsonencode](https://developer.hashicorp.com/terraform/language/functions/jsonencode) to encode arrays or objects. Learn about how to [define the action payload](https://docs.getport.io/create-self-service-experiences/setup-backend/#define-the-actions-payload).
```
Comment (erikzaadi) [inline docs/resources/port_action.md:263]: This sounds better than the shorter explanations above, still duplicate link

File: docs/resources/port_blueprint_permissions.md [hunk 4]
```diff
@@ -7,63 +7,64 @@ description: |-
   Docs about blueprint permissions can be found here https://docs.getport.io/build-your-software-catalog/set-catalog-rbac/examples/#setting-blueprint-permissions
   Example Usage
   Allow access to all members:
-  hcl
-  resource "port_blueprint_permissions" "microservices_permissions" {
-    blueprint_identifier = "my_blueprint_identifier"
-    entities= {
+  ```hcl
+  resource "portblueprintpermissions" "microservicespermissions" {
```
Comment (erikzaadi) [inline docs/resources/port_blueprint_permissions.md:11]: Was this formatted by mistake?

There are missing `_`'s

**Review by erikzaadi — APPROVED** [review]
Approving to save ping pong, some small [NITBITS]

File: docs/resources/port_action.md (hunk 1 above)
Comment (Tankilevitch) [reply docs/resources/port_action.md:226]: yes

**Pull request merged**